src/v/kafka/client/configuration.cc

```diff
@@ -105,12 +105,20 @@ configuration::configuration()
       "Interval (in milliseconds) for consumer request timeout",
       {},
       100ms)
+  , consumer_request_min_bytes(
+      *this,
+      "consumer_request_min_bytes",
+      "Min bytes to fetch per request",
+      {},
+      1,
+      {.min = 0})
```

Comment:
considering the `should_stop_fetch()` condition, it seems like a `min_bytes: 0` is not great, but probably nbd considering that we've been living with this "forever" and the default is 1.

Reply:
Looking at, for example, lidbrdkafka config, 0 is valid for both, so I took inspiration from there.

Reply:
nice, makes sense